src/styles/index.css

```diff
@@ -0,0 +1,203 @@
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__page {
+  margin: 0;
+  cursor: pointer;
+}
+
+.calendar__element {
+  display: flex;
+  margin: 10px;
+  gap: 1px;
+  width: 707px;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0;
+}
+
+.calendar__block {
+  width: 100px;
+  height: 100px;
+  text-align: center;
+  line-height: 100px;
+  border: 1px black;
```

Comment:
The `border` property is missing a style value (e.g., `solid`). It should be something like `border: 1px solid black;`.